### PR TITLE
CI script to run unit test

### PR DESCRIPTION
**Pull request by am2089**
# CI script to run unit test
* Closes #issue 53
* Triggers CI script to run unit test manually

# How I Tested
Can't be tested until accepted by @mikaelacaron

### CODE REVIEW

File: .github/workflows/unit-test ci.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+name: Basic-Car-Maintenence CI
```
Comment (mikaelacaron) [inline .github/workflows/unit-test ci.yml]:
```suggestion
name: Run Unit Tests
```

File: .github/workflows/unit-test ci.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+    - name: Build and test
+      run: |
+          xcodebuild clean build test -project Basic-Car-Maintenance.xcodeproj -scheme Basic-Car-Maintenence -destination 'platform=iOS Simulator, name=iPhone 15'
+      
+      
+      
+
+
+
```
Comment (mikaelacaron) [inline .github/workflows/unit-test ci.yml:34]: Next time, you can remove the extra line breaks

**Review by mikaelacaron — APPROVED** [review]
Great work! Going to merge it and see if it works!
Notice the suggestions I made, I'll merge these, but in the future you can click "Commit suggestion" to directly use the suggestions I have in the PR

File: .github/workflows/unit-test ci.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      - main
+      - dev
+  pull_request:
+    branches:
+      - main
+      - dev
+
+jobs:
+  build:
+    runs-on: macos-14.0 or later
```
Comment (mikaelacaron) [inline .github/workflows/unit-test ci.yml]:
```suggestion
    runs-on: macos-13.5 or later
```

**Comment by mikaelacaron** [comment]
I'm waiting to see if the CI runs properly before merging

**Comment by am2089** [comment]
Ohh ok, I will take your advice into account for next time. So can I look at other issues now since this one is approved?On Oct 7, 2023, at 10:34 PM, Mikaela Caron ***@***.***> wrote:﻿
@mikaelacaron approved this pull request.

Great work! Going to merge it and see if it works!
Notice the suggestions I made, I'll merge these, but in the future you can click "Commit suggestion" to directly use the suggestions I have in the PR

In .github/workflows/unit-test ci.yml:
> @@ -0,0 +1,34 @@
+name: Basic-Car-Maintenence CI

⬇️ Suggested change
-name: Basic-Car-Maintenence CI
+name: Run Unit Tests





In .github/workflows/unit-test ci.yml:
> +      
+      
+
+
+

Next time, you can remove the extra line breaks

**Comment by mikaelacaron** [comment]
@am2089 Actually I don't think this works, and looking at your [own fork](https://github.com/am2089/Basic-Car-Maintenance/actions/workflows/unit-test%20ci.yml), it doesn't look like it works on yours either

**Review by mikaelacaron — CHANGES_REQUESTED** [review]
I don't think this works

**Comment by am2089** [comment]
Ok I will go back and see what’s wrong.Sent from my iPhoneOn Oct 7, 2023, at 11:35 PM, Mikaela Caron ***@***.***> wrote:﻿
@mikaelacaron requested changes on this pull request.

I don't think this works

**Comment by am2089** [comment]
Hey I think it will work now, there was a typo in the scheme name.On Oct 7, 2023, at 11:35 PM, Mikaela Caron ***@***.***> wrote:﻿
@mikaelacaron requested changes on this pull request.

I don't think this works

**Comment by mikaelacaron** [comment]
Replaced with #98

**Pull request closed without merging**